src/Compilers/Core/Portable/Compilation/NullableContextOptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents the default state of nullable analysis in this compilation.
+    /// </summary>
+    [Flags]
+    public enum NullableContextOptions
+    {
+        /// <summary>
+        /// The nullable analysis feature is disabled.
+        /// </summary>
+        Disable = 0b00,
```

Comment:
>0b00 [](start = 18, length = 4)

Personally I prefer `1<<0` and `1<<1` and `1<<2` etc.

Reply:
I find the binary prefix format, I find it more readable as I don't have to do any mental bit shifting. Everything is laid out exactly as intended.